packages/idos-sdk-js/README.md

```diff
@@ -325,6 +325,30 @@ There are two things that a dApp needs to have setup:
 >
 > Make sure you don't lose access to either secret keys. Otherwise, you won't be able to authenticate or decrypt credential contents. The idOS team won't be able to help you.
 
+### Write Grants
```

Comment:
I don't think I want this file to grow any bigger, come to think of it. :/

This README has turned into a dApp integration guide, and not really a README. Furthermore, it also tries to be an implementation reference at the end of the guide.

I think that just pilling on more sections on this document is starting to feel like debt. The need to extract some parts away (and divide it into, at least, "Guides", "Reference implementations", and "Glossary") is something that I'm feeling more and more.

I know this is not actionable in this formulation, but I wanted to preliminarily share my thoughts.